package/network/services/samba36/files/samba.init

```diff
@@ -5,7 +5,6 @@ START=60
 USE_PROCD=1
 
 smb_header() {
-	local interface
 	config_get interface $1 interface "loopback lan"
```

Comment:
this is also fragile. you are moving the interface variable to the global scope as you need id when settign up the triggers. this will work as long as procd.sh does not have a variable called interface. please rename the variable to samba_ifaces or something similar